Libplanet.Headless/ReducedStore.cs

```diff
@@ -16,7 +16,7 @@ namespace Libplanet.Headless
     /// <list type="bullet">
     /// <item><description><see cref="PutTxExecution(TxSuccess)"/></description></item>
     /// </list>
-    /// </summary>
+    /// </summary>.
```

Comment:
Does this `.` mean anything?

Reply:
~~It is `dotnet format`'s act 🙄~~ I'll revert it. Thanks 🙏🏻

Reply:
Amended! https://github.com/planetarium/NineChronicles.Headless/compare/abd3880e5025d5c36a24760f6f0f5ba0a96c4cdf..fee345b9d7540483a27a1b9c00843893ad20e6f6